git.md

```diff
@@ -211,17 +231,17 @@ The relevant commits are all rewritten to be brought on [top of the `HEAD` of th
 
 ## On Reviewing
 
-Try to set aside some quality time as a routine to proactively review others' pull/merge requests, as thoroughly as your bandwidth allows. If there's no time, and/or no willingness, do not just cast approval without reviewing.
+Try to set aside some quality time as a routine to proactively review others' pull/merge requests, as thoroughly as your bandwidth allows. If there's no time, and/or no willingness, do not just cast approve without reviewing. For GitHub repositories, this is the perfect time to make good use of [GitHub's notifcation with a `review-requested` filter](https://github.com/notifications?query=reason%3Areview-requested).
```

Comment:
suggestions
- should be 'do not just approve without reviewing'
- typo in 'Github's notification'